dev-docs/bidders/adinify.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: bidder
+title: Adinify
+description: Adinify Bidder Adapter
+biddercode: adinify
+aliasCode : smarthub
+usp_supported: true
+coppa_supported: true
+schain_supported: true
+dchain_supported: true
+media_types: banner, video, native
+safeframes_ok: true
+deals_supported: true
+floors_supported: true
+fpd_supported: false
+pbjs: true
+pbs: true
+pbs_app_supported: true
+multiformat_supported: true
```

Comment:
Please use one of the pre defined values: https://docs.prebid.org/formats/multiformat.html

```suggestion
multiformat_supported: will-bid-on-any
```

Or `will-bid-one-one` if there's a priority on your end.

Reply:
fixed to will-bid-on-any